modules/element.py

```diff
@@ -100,12 +111,14 @@ def __str__(self):
             Returns element, isotope and amount in string format.
         """
         if self.isotope and self.amount:
```

Comment:
There's no case for `self.isotope`, `self.amount` and `self.sequence` (and other combinations with `self.isotope`). Also, there's a missing space for `self.amount`.

Reply:
Okay. Why there is no case for them?

Reply:
Because you haven't added them.

Reply:
Oh! You mean they are missing. But do what you actually mean when you say that there's no case?  Aren't there `elif isotope`, `elif amount`, `elif sequence` etc. What am I missing here?

Reply:
The `__str__` method must handle different combinations from the largest to the smallest (something like this):
- isotope & amount & sequence
- isotope & amount
- isotope & sequence
- amount & sequence
- isotope
- amount
- sequence
- (none)

Problem:
```
ele = Element("He", isotope=2, amount=1.0, sequence=3)
print(ele)  # Should print "2He#3 1.0", but prints "2He1" or something like that
```

The `sequence` is only handled if there is no `isotope` or `amount` set.

`from_string` works (mostly?) correctly, do a similar if-else thing here.

Reply:
There's also something wrong with the `amount` in general. It should be a float, but it's an int. Look into that too.